src/main/fc/rc_modes.h

```diff
@@ -92,6 +92,7 @@ typedef struct modeActivationCondition_s {
     boxId_e modeId;
     uint8_t auxChannelIndex;
     channelRange_t range;
+    uint8_t modeLogic;  // 0 = OR, 1 = AND
```

Comment:
This should use an enum instead of a comment.

Reply:
Done